package.json

```diff
@@ -24,57 +24,57 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@material-ui/core": "^4.4.3",
-    "@material-ui/styles": "^4.4.3",
+    "@material-ui/core": "^4.5.2",
+    "@material-ui/styles": "^4.5.2",
     "connected-react-router": "^6.5.2",
     "history": "^4.10.1",
-    "immer": "^4.0.0",
+    "immer": "^5.0.0",
     "normalizr": "^3.4.1",
     "prop-types": "^15.7.2",
-    "react": "^16.9.0",
-    "react-dom": "^16.9.0",
+    "react": "^16.11.0",
+    "react-dom": "^16.11.0",
     "react-error-boundary": "^1.2.5",
-    "react-hot-loader": "^4.12.14",
+    "react-hot-loader": "^4.12.15",
     "react-redux": "^7.1.1",
-    "react-router-dom": "^5.0.1",
+    "react-router-dom": "^5.1.2",
     "redux": "^4.0.4",
     "redux-devtools-extension": "^2.13.8",
     "redux-thunk": "^2.3.0",
     "strange-middle-end": "^0.1.1",
```

Comment:
I think this is calling for a bump to `^0.2.0` now!

Reply:
That's true!

```suggestion
    "strange-middle-end": "^0.2.0",
```